# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: media/mtransport/ipc/WebrtcTCPSocket.h
# Commit: 81ab3774184b
# Full Hash: 81ab3774184b5f52eda50608e0a311f873df78af
# Author: Byron Campen [:bwc] <docfaraday@gmail.com>
# Date: 2019-09-24 03:42:17
# Description:
#   Bug 1582646: Cancel proxy lookup when socket closes. r=mjf
#   
#   Also, since OnProxyAvailable always happens on main regardless of the target we
#   set, don't bother setting the target.
#   
# ==============================================================================

diff -r c8bbd3388d4e -r 81ab3774184b media/mtransport/ipc/WebrtcTCPSocket.h
--- a/media/mtransport/ipc/WebrtcTCPSocket.h	Mon Sep 23 18:15:58 2019 +0000
+++ b/media/mtransport/ipc/WebrtcTCPSocket.h	Fri Sep 20 15:13:44 2019 +0000
@@ -97,6 +97,7 @@
   nsCOMPtr<nsIAsyncOutputStream> mSocketOut;
   nsCOMPtr<nsIEventTarget> mMainThread;
   nsCOMPtr<nsIEventTarget> mSocketThread;
+  nsCOMPtr<nsICancelable> mProxyRequest;
 };
 
 }  // namespace net
